emcc.py

```diff
@@ -1430,6 +1430,7 @@ def has_c_source(args):
       shared.Settings.EXPORTED_FUNCTIONS += ['_malloc', '_free']
 
     if shared.Settings.WASM_BACKEND:
+      shared.Settings.EXPORTED_FUNCTIONS += ['_stackSave', '_stackRestore', '_stackAlloc']
```

Comment:
Do these need to stay exported? Aren't they exposed to JS runtime (or user?) code?